assets/dev/js/editor/views/base-container.js

```diff
@@ -74,6 +74,8 @@ module.exports = Marionette.CompositeView.extend( {
 			options.onBeforeAdd();
 		}
 
+		this.applyChildContentWidthSettings( newItem );
```

Comment:
```suggestion
		if ( this.filterSettings ) {
	                this.filterSettings( newItem );
                 }
```
This method `applyChildContentWidthSettings` is specific to our `container` element/widget and does not belong in the base class.

I would simply add a method (maybe named) `filterSettings` to `elementor/assets/dev/js/editor/elements/views/container.js` and then it would be called from this base class if it exists. This way it is generic and can be added to anything that extends this class like sections/columns/containers.